javascript-source/commands/customCommands.js

```diff
@@ -441,6 +441,21 @@
             }
         }
 
+        if (message.match(/\(repeat\s[1-9]\d*,[\w\s]+\)/g)) {
+            var matches = message.match(/\(repeat\s([1-9]\d*),([\w\s]+)\)/);
+            var allMatch = matches[0];
+            var counter = matches[1];
```

Comment:
Making sure to include a limit here, I would say 10 at max so people don't run into Twitch's chat limit.

Reply:
I would not set a limit if a user wants to enter a high value so why not. I currently use 30 myself at its works fine with emots for me. So if its necessary whats about 30?

Reply:
https://github.com/PhantomBot/PhantomBot/blob/master/development-resources/CODESTYLE.md#chat-limits